cremalab/workout-tracker
# Feature profile page

## Reference
- Original [Asana Task](https://app.asana.com/0/622910146909793/622910146909836)
- Related Asana Tasks
--[Add Profile Pic](https://app.asana.com/0/622910146909793/622910146909840)
--[Edit Profile Pic](https://app.asana.com/0/622910146909793/622910146909842)
--[Delete Profile Pic](https://app.asana.com/0/622910146909793/622910146909844)
--[Save workouts with user id](https://app.asana.com/0/622910146909793/679305561011534)

## Overview

This PR allows users to create a user profile and upload a profile picture.

### Notes

In the future might try to create own upload tool versus using Cloudinary widget. Also need to restrict Profile page so that unauthenticated users can't view it.

## To-Do

- [x] Update profile pic to populate based off of redux state, not local state and to return default pic if none provided.
- [x] Save user input in form to the DB
- [x] Display profile info from DB for logged in user

## Testing Instructions

-npm install && cd client && npm install && cd ..
-npm start
-http://localhost:3000/
-Sign up with new user then login
-Navigate to Profile page and click icon to upload new pic
-New pic should appear after uploading
-Going back to login screen and logging in again with same user should return the same pic you just loaded
